VnE/Source/Extensions/ChemPack/applyCls.py

```
@@ -17,21 +17,22 @@ def execute():
     def payload(mol_tuple):
 
         def fracToDec(a, b, c, al, be, ga, coords):
```

Ruff (0.14.14)

[warning] Missing return type annotation for private function `fracToDec`

(ANN202)